frontend/__tests__/setup-test-env.ts

```diff
@@ -1,7 +1,13 @@
 import { JSDOM } from 'jsdom';
+import { afterEach } from 'vitest';
 
 // Workaround: For some reason FormData is not set to jsdom's by default
 const jsdom = new JSDOM(`<!doctype html>`);
 const { FormData } = jsdom.window;
 window.FormData = FormData;
 global.FormData = FormData;
+
+afterEach(() => {
+  vi.clearAllMocks();
+  vi.resetAllMocks();
+});
```

Comment:
1. doesn't `resetAllMocks()` also clear all mocks?
2. While I'm not totally opposed to this, wouldn't it make more sense for the creator of the mocks to also control their lifecycles? What if a test needed a mock's state to persist across multiple tests? (and yes, I acknowledge this is probably bad practice)